lib/pcap.cmake

```diff
@@ -35,17 +39,20 @@ if (BUILD_PCAP_LIB AND NOT (BUILD_ONLY_DOCS))
     find_library(LIBPCAP_LIB NAMES libpcap.a pcap PATHS "${LIBPCAP_LIB_DIR}" NO_DEFAULT_PATH)
   endif ()
 
-  # static pcap needs -libverbs -lnl-genl-3 -lnl-3  -ldbus-1
+  # static pcap needs -lnl-genl-3 -lnl-3 # maybe openssl and crypto too???
```

Comment:
As far as I know, pcap shouldn't depend on these libraries.

Reply:
After you removed these lines in https://github.com/nqminds/EDGESec/pull/106/commits/99bedafa65c5519207408b79504df4cf1f59d3fa, compiling still worked on my PC.

However, my `./build/lib/pcap/lib/pkgconfig/libpcap.pc` file says `Libs.private:  -lnl-3 -l/usr/lib/x86_64-linux-gnu/libssl.so -l/usr/lib/x86_64-linux-gnu/libcrypto.so`, so it should need `lnl-3` `libssl` and `libcrypto` for static linking.

Looking at `libpcap`'s build file, we can see that if we're compiling `libpcap` for Linux, it will automatically build netlink support, see https://github.com/the-tcpdump-group/libpcap/blob/master/CMakeLists.txt#L1331-L1348

Maybe PCAP does require linking to `nl-3`, but we're already linking it somewhere?? Not sure.

Reply:
If you're sure that PCAP will never need `netlink` support, I'll compile `libpcap` with `BUILD_WITH_LIBNL=OFF` so that we never have to worry `netlink` support.

Reply:
Lets compile it with `BUILD_WITH_LIBNL=OFF` for now.